moped-editor/src/views/projects/projectView/CommentInputQuill.js

```diff
@@ -68,6 +68,8 @@ const CommentInputQuill = ({
               </div>
             )}
             <ProjectSaveButton
+              // disable save button if no text after removing html tags
+              disabled={!noteText.replace(/(<([^>]+)>)/gi, "")}
```

Comment:
This is such an edge case, but the string `<>` would get past this regex and come out the otherside still as `<>`. I also think the parentheses are unneeded since you are not capturing the matched values. I think you could go with `/<[^>]*>/g`. I read it as: "This matches the situation where there is content in between a less-than and greater-than symbol, and when that content is anything that does not include a greater-than symbol, including the null set." What do you think? That also drops the `i` flag off there since there isn't any case-sensitive text being considered.


![image](https://user-images.githubusercontent.com/3653206/208800071-871e7b77-f1d9-4e6f-8a5b-69eafb6c5bfd.png)